docker-compose.yml

```diff
@@ -14,17 +14,19 @@ services:
     ports:
       - "3000:3000"
     environment:
-      SOLR_URL: "http://solr:8983/solr/blacklight-core"
-      DEVELOPMENT_ENVIRONMENT: "docker"
+      - SOLR_URL # Set via environment variable or use default defined in .env file
     entrypoint:
       - /app/bin/start.sh
 
   solr:
-    image: solr:7
+    environment:
+      - SOLR_PORT # Set via environment variable or use default defined in .env file
+      - SOLR_VERSION # Set via environment variable or use default defined in .env file
+    image: "solr:${SOLR_VERSION}"
     volumes:
       - $PWD/lib/generators/blacklight/templates/solr/conf:/opt/solr/conf
     ports:
-      - "8983:8983"
+      - "${SOLR_PORT}:8983"
```

Comment:
I really like the use of SOLR_PORT environment variable here.

Reply:
Thanks!